src/Authentication/Login.php

```diff
@@ -202,11 +202,12 @@ private function getObjectFromResponse(ResponseInterface $rawResponse): \stdClas
     private function getLoginRequest(): RestRequest
     {
         $config = $this->authentication['loginRequest'];
+        $fnContext = LoginAuthLoginRequestContext::create($this->configAttributes);
         if (!empty($config['params'])) {
-            $config['params'] = $this->buildLoginRequestFunctions($config['params']);
+            $config['params'] = UserFunction::build($config['params'], $fnContext);
```

Comment:
Este jeden fix k `Login`, vsimol som si, ze sa tam rovnaky `context` vytvara zbytocne 2x, tak som to fixol.